tests/common.go

```diff
@@ -493,7 +500,7 @@ func runCmd(cmd string, n node.Node) {
 // PerformSystemCheck check if core files are present on each node
 func PerformSystemCheck() {
 	context(fmt.Sprintf("checking for core files..."), func() {
```

Comment:
not related to your change but the description here can be improved to 

`To check for core files`

`verify if cores files are present on any node`